doc/source/whatsnew/v0.22.0.txt

```diff
@@ -140,7 +140,7 @@ Sparse
 Reshaping
 ^^^^^^^^^
 
--
+- :func:`get_dummies` now supports ``dtype`` argument
```

Comment:
add a little more expl, add the PR number as the issue number. Move to Other Enhancements section.